manifests/init.pp

```diff
@@ -175,6 +178,7 @@
   Integer[0] $redis_db = 8,
   Stdlib::Fqdn $servername = $facts['networking']['fqdn'],
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
+  Array[Stdlib::Absolutepath] $allowed_export_path = ['/var/lib/pulp/exports'],
```

Comment:
```suggestion
  Array[Stdlib::Absolutepath] $allowed_export_path = [],
```
My thinking is that we should try as best as possible to stick with defaults within this module that would make sense for a user who just wants to install pulp3 with puppet.

Since we'll expose this parameter in puppet-FPC and in the installer answers files for Katello and FPC scenarios, those are more appropriate places to set values that make sense specifically for Katello / content proxy.